mmv1/products/cloudidentity/Group.yaml

```diff
@@ -128,6 +128,36 @@ properties:
     description: |
       An extended description to help users determine the purpose of a Group.
       Must not be longer than 4,096 characters.
+  - !ruby/object:Api::Type::Array
+    name: 'additionalGroupKeys'
+    output: true
+    description: 'Additional group keys associated with the Group'
+    item_type: !ruby/object:Api::Type::NestedObject
+      properties:
+        - !ruby/object:Api::Type::String
+          name: 'id'
```

Comment:
I added to the [reference doc](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/api/type.rb#L40-L41) very recently, but I think these nested fields should be `output: true` as well, so that they are indicated properly in the docs.

However, I'm not sure I understand how this field works, because I also see `immutable: true` on `namespace`, which looks to be a configurable field?

Reply:
Ah sorry, very sloppy copy-pasting on my part 🤦

Reply:
Fixed in [dbe6766](https://github.com/GoogleCloudPlatform/magic-modules/pull/9217/commits/dbe6766f24eb67bb5c7eb23c7ba9c275d0f1189a) - how they're all explicitly output field and I removed the immutable property that shouldn't have been there at all